code/mission/missionparse.cpp

```diff
@@ -5498,6 +5522,9 @@ void parse_one_background(background_t *background)
 		// add it
 		background->bitmaps.push_back(sle);
 	}
+
+	// always set the flag when we're done parsing, because the angles are now stored correctly and we want to save them correctly
+	background->flags.set(Starfield::Background_Flags::Corrected_angles_in_mission_file);
```

Comment:
This line confuses me. It makes it sound like the corrected angles have been saved to the mission file, when what we've actually done is correct the background's angles that are stored in memory.

Reply:
We have corrected the angles that are stored in memory, yes.  Now we need to choose whether to save them correctly or save them incorrectly, so the default is to save them correctly.  But this can be changed in the background editor.